flamedisx/lxe_blocks/raw_signals.py

```diff
@@ -80,72 +69,50 @@ def _compute(self,
             loc=mean, scale=std + 1e-10
         ).prob(s_observed)
 
+        ''' Think can also remove this chunk
```

Comment:
Yes, I don't think you'd want an acceptance on a non-measured signal

Reply:
Done